### DISCUSSION TITLE
How to close "Compare&Pull request" warning

### DISCUSSION THREAD

**Discussion by FizzJu** [community/community#11777] [Pull Requests]
I forked Unreal Engine for learning.
I follow their tutorial to create for my own edit.
But the "Compare & Pull request" warning is bothering, because I have no need to commit to Epic Games
Even though I want to contribute to it. But for now, I just a beginner.
***Does it have any way to close “Compare&pull request” warning  ***

↓↓↓↓ You can translate this block (Above maybe no good for understand)
我从 EpicGames 的 UnrealEngine 拷贝了分支去学习引擎代码
我按教程创建了分支，引擎本身的版本就很多，为了不让引擎原生的代码，和学习的代码冲突。
我按照EpicGames的教程去创建了分支，但是创建分支后“Compare&pull request”的警告非常的烦人和碍事
因为那些分支没有提交的必要
***有没有什么方法能关闭 “Compare&pull request” 的警告  ***
这个分支我不会提交任何代码到EpicGames。因为这个只是为了学习用的，
并不能修正任何引擎的漏洞，即使我以后有计划为UnrealEngine的开源代码做出，但是现在我只是个初学者。

**Comment by Zi-Tao** [comment]
@DisVoider  You can try to close a pull request!
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/closing-a-pull-request

**Comment by lyc452** [comment]
那就用ublock origin之类的插件给他屏蔽了得了
